disable_prelinking.cf

```diff
@@ -29,15 +29,15 @@ bundle agent disable_prelinking
 
   files:
     # Disable prelink in configuration file
+    _stdlib_path_exists_prelink::
       "$(_config_file)" -> { "CCE-27078-5" }
-        create => "true",
```

Comment:
Hmm, I'd actually keep this `create => "true"`. If the binary exists and there's no config, the policy should create it to make sure prelinking is disabled.